### PR TITLE
#161477688 Update TravisCI to run tests

### PR DESCRIPTION
**Pull request by mrSerious**
#### What does this PR do?

Update TravisCI to run tests

#### Description of Task to be completed?

- add a script to run tests on TravisCI

#### How should this be manually tested?

N/A

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#161477688](https://www.pivotaltracker.com/story/show/161477688)

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+services:
+  - postgresql
 cache:
   directories:
-    - "node_modules"
\ No newline at end of file
+    - "node_modules"
+script:
+  - npm test
+
```
Comment (KvNGCzA) [inline .travis.yml]: remove the tab character at the end of the file

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
   - "stable"
+services:
+  - postgresql
 cache:
   directories:
-    - "node_modules"
```
Comment (KvNGCzA) [inline .travis.yml]: create an extra line at the end of the file

Comment (mrSerious) [reply .travis.yml]: I have actioned your feedback, Chris. Everything should be good now.

**Comment by Darthrighteous** [comment]
Could you explain why you've added this script to the `.travis.yml` file? is it doing anything new?

**Comment by mrSerious** [comment]
> @mrSerious Could you please explain why you've added this script to the `.travis.yml` file? is it doing anything new?

I like to include the command to run test explicitly, do you think I should take it out?

**Comment by Darthrighteous** [comment]
> > @mrSerious Could you please explain why you've added this script to the `.travis.yml` file? is it doing anything new?
> 
> I like to include the command to run test explicitly, do you think I should take it out?
No, I don't. I was just wondering if it was there for any other reason.

**Pull request merged**